sysconfig.spec.in

```diff
@@ -76,23 +82,29 @@ Provides:       /sbin/netconfig
 
 %description netconfig
 This package provides the netconfig scripts to apply network
-provided settings like DNS or NIS into system files.
+provided settings like DNS, NIS or NIS into system files.
```

Comment:
typo?
```suggestion
provided settings like DNS, NTP or NIS into system files.
```

Reply:
yes, this was the intention to add the missed ntp...